embedding/cpp/flutter_app.cc

```diff
@@ -72,6 +72,10 @@ void FlutterApp::OnRegionFormatChanged(app_event_info_h event_info) {
 }
 
 int FlutterApp::Run(int argc, char **argv) {
+  for (int i = 0; i < argc; i++) {
+    dart_entrypoint_args_.push_back(argv[i]);
```

Comment:
Why don't you make a setter like a `SetDartEntrypoint`?

Reply:
`dart_entrypoint_args_` (which will be changed to a private member in https://github.com/flutter-tizen/flutter-tizen/pull/388) is implicitly set by the embedding and users are not expected to access the value.

Reply:
Of course we can implement it, if we need.

Reply:
I don't think it's necessary right now.